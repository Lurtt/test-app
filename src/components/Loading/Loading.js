import React from 'react'
import styled from 'styled-components'

import svg from './three-dots.svg'

const LoadingStyle = styled.img`
  display: flex;
  margin: auto;
`

export default function Loading() {
  return <LoadingStyle src={svg} />
}
