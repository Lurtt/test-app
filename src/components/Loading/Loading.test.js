import React from 'react'
import renderer from 'react-test-renderer'

import Loading from './'

jest.mock('./three-dots.svg', () => 'SVG_CONTENT_HERE')

describe('<Loading />', () => {
  test('Should render', () => {
    const result = renderer.create(<Loading />).toJSON()
    expect(result).toMatchSnapshot()
  })
})
