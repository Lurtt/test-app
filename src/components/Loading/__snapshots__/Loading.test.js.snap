// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> Should render 1`] = `
<img
  className="sc-bdVaJa hzhSHT"
  src="SVG_CONTENT_HERE"
/>
`;
